web/static/js/event_chat.js

```diff
@@ -3,10 +3,11 @@ class EventChat {
     socket.connect()
     this.socket = socket
     const el = document.getElementById(element)
+    const eventId = el.getAttribute("data-event-id")
```

Comment:
Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)